src/container/Message/Message.tsx

```diff
@@ -0,0 +1,16 @@
+import { FC } from "react";
+
+type Props = {
+  color?: string;
+  message: string;
+};
+
+const Message: FC<Props> = ({ message, color = "bg-red-500" }) => {
```

Comment:
Default is error but could be used for other types in teh future or why the change to `Message` only?

Reply:
Yes, I'm using the message component for the yellow message as well now.

Before it was just an ErrorMessage, hence the name.